### DISCUSSION TITLE
Failed to pull latest Vector chart release (v0.38.0)

### DISCUSSION THREAD

**Discussion by jszwedko** [artifacthub/hub#4175] [General]
Hi all,

With Vector's [latest Helm chart release](https://github.com/vectordotdev/helm-charts/releases/tag/vector-0.38.0), Artifacthub failed to pull it with:

```
error preparing package: error loading chart (https://github.com/timberio/helm-charts/releases/download/vector-shared-0.1.0/vector-shared-0.1.0.tgz): Get "https://github.com/vectordotdev/helm-charts/releases/download/vector-shared-0.1.0/vector-shared-0.1.0.tgz": context deadline exceeded (Client.Timeout exceeded while awaiting headers) (package: vector-shared version: 0.1.0)
error preparing package: error loading chart (https://github.com/vectordotdev/helm-charts/releases/download/vector-0.38.0/vector-0.38.0.tgz): Get "https://github.com/vectordotdev/helm-charts/releases/download/vector-0.38.0/vector-0.38.0.tgz": context deadline exceeded (Client.Timeout exceeded while awaiting headers) (package: vector version: 0.38.0)
(next check in ~ 28 minutes)
Last security scan: 20 hours ago(next scan in ~ 13 minutes)
```

This looks like a transient network issue since the URLs work for me. I expected it to pull successfully during the next scheduled check, but it's been in this state for about two days now.

Would someone be able to take a peek at this for us? Thank you!

**Comment by tegioz** [comment]
Hi @jszwedko 👋

If we can read the repository index successfully, but something goes wrong fetching the charts packages, we don't try to process them again until we detect something has changed in the repository (based on a hash of the `index.yaml` file, excluding the `generated` field). We do it this way to avoid processing repositories over and over when they may have a problem, as we have thousands of them to process periodically.

I've forced your repository to be processed and everything should be fine now. You can trigger this processing on your side by updating an existing package or releasing a new version (maybe we'll add an option to the control panel to make this easier).

**Reply by jszwedko** [reply]
Thank you @tegioz ! I can verify that it has been fetched now. Having an option in the control panel to force reprocessing would definitely be helpful. For now, we can plan to make a trivial change to `index.yaml` if this happens again.
